spec/models/claim/advocate_claim_spec.rb

```diff
@@ -1553,6 +1553,44 @@ def expect_has_authorised_state_to_be(bool)
       expect(claim.expenses).to have(1).member
       expect(claim.expenses_total).to eq 40.0
     end
+
+    describe '#agfs_reform_discontinuance?' do
```

Comment:
question over whether this method is applicable at all from a business perspective. I thought it applies to  scheme 9  too (and need to check LGFS scheme 8 impact)

Reply:
It does - had this confirmed by Angela.